.github/workflows/pr-realm-js.yml

```diff
@@ -294,8 +296,15 @@ jobs:
       - name: Install pods
         if: ${{ matrix.variant.environment == 'react-native' && matrix.variant.os == 'ios'}}
         working-directory: integration-tests/environments/${{ matrix.variant.environment }}
+        env:
+          USE_HERMES: ${{ matrix.variant.engine == 'hermes' && '0' || '1' }}
```

Comment:
Why '0' || '1'?

Reply:
Because we want `USE_HERMES` to equal `1` when engine is 'hermes' and GitHub Actions doesn't have a ternary operator. But actually - looking at this again, I messed up the order 🤦